examples/Filter response content based on product name.policy.xml

```diff
@@ -15,10 +15,10 @@
   <outbound>
     <base />
     <choose>
-      <when condition="@(context.Response.StatusCode == 200 && context.Product.Name.Equals("Starter="""))">
+      <when condition="@(context.Response.StatusCode == 200 && context.Product.Name.Equals("Starter"))">
         <set-body>
           @{
-            <!-- NOTE that we are not using preserveContent=true when deserializing response body stream into a JSON object since we don't intend to access it again. See details on https://docs.microsoft.com/en-us/azure/api-management/api-management-transformation-policies#SetBody -->
+            //NOTE that we are not using preserveContent=true when deserializing response body stream into a JSON object since we don't intend to access it again. See details on https://docs.microsoft.com/en-us/azure/api-management/api-management-transformation-policies#SetBody
```

Comment:
This is not valid XML, can you use `<!-- -->` please?

Reply:
Thanks @mike-urnun-msft !

Reply:
@tomkerkhove sure, updated!